modules/javafx.web/src/main/native/Source/WebCore/platform/graphics/java/MediaPlayerPrivateJava.cpp

```diff
@@ -430,6 +430,12 @@ float MediaPlayerPrivate::currentTime() const
         LOG_TRACE1("MediaPlayerPrivate currentTime returns (seekTime): %f\n", m_seekTime);
         return m_seekTime;
     }
+
+    // in case of hls media m3u8 format check network state
+    // since jfx media do not support hls live streaming protocol
+    if ( MediaPlayerNetworkState::NetworkError == MediaPlayer::NetworkState::NetworkError)
```

Comment:
Are there other errors that might occur, or is this the only one you need to check?

Minor: delete the extra space after the `(`.

Reply:
done

Reply:
The answer to my question is still pending. Please take a look at whether or not this is the only error you need to check. If it is, can you say why?

Reply:
Once the Player is in an error state, play, pause, seek, etc. all of these operations would not be called. Only the current time which is the last played point of time needs to be updated to the player user interface.

Reply:
OK. My question then is whether `MediaPlayerNetworkState::NetworkError` is the only error condition you need to consider.